modules/ossm-security-cert-manage.adoc

```diff
@@ -70,7 +70,7 @@ $ RATINGSPOD=`oc get pods -l app=ratings -o jsonpath='{.items[0].metadata.name}'
 +
 [source,terminal]
 ----
-$ oc exec -it $RATINGSPOD -c istio-proxy -- /bin/cat /etc/certs/root-cert.pem > /tmp/pod-root-cert.pem
+$ oc exec -it $RATINGSPOD -c istio-proxy -- /bin/cat /var/run/secrets/istio/root-cert.pem > /tmp/pod-root-cert.pem
 ----
```

Comment:
line 32, @rcernich Should we replace `The default `rootCADir` is `/etc/cacerts` with `The default `rootCADir` is `/var/run/secrets/istio`  ?

Reply:
line 80, @rcernich I don't see that file  `oc exec -it $RATINGSPOD -c istio-proxy -- /bin/cat /etc/certs/cert-chain.pem > /tmp/pod-cert-chain.pem` in istio-proxy container. Do you know what's the correct file path ?

Reply:
I think it's version specific.  /etc/certs is for 1.x, /var/run/secrets/istio is for 2.x.

Reply:
So since this is under the v2x directory, it should be updated to /var/run/secrets/istio as in the PR? (Not familiar with this product area at all, so wanna make sure I'm following :slightly_smiling_face: )

Reply:
In v2x SMCP, we should replace line 32 and line 80 /etc/certs with /var/run/secrets/istio . If self-signed certs are used, there is no cert-chain.pem file.

Reply:
Thanks, @yxun - a couple questions:
1. Line 32 has `/etc/cacerts`, not `/etc/certs`. Just want to verify with you that I should replace with `/var/run/secrets/istio`
2. Line 47 also mentions `/etc/cacerts` - do I need to do anything with this?
3. If there is no `cert-chain.pem` file, do I replace `/etc/certs/cert-chain.pem` with only `/var/run/secrets/istio`? What happens to `/tmp/pod-cert-chain.pem` in that same command?

Reply:
@rcernich could you help review and confirm the correct cert file location ? I don't know the default values of those two files in 2.x template.

Reply:
`/etc/cacerts` is for configuring the certs used by pilot for signing.  This would be used for configuring a custom private key for pilot to use for signing (as opposed to a self generated, self-signed key).  `/etc/certs` is where the certs are mounted in the proxy sidecar.

Reply:
Thanks, rcernich. @jeana-redhat This PR is good now. I will approve my review. Thanks.

Reply:
Thank you both very much!